### PR TITLE
Add MongoDB Analyzer

### PR DESCRIPTION
**Pull request by biniona-mongodb**
Add MongoDB Analyzer to the Develop Applications tab.

JIRA: None

  Staging: https://docs-mongodborg-staging.corp.mongodb.com/5f065e5/landing/docsworker-xlarge/Add-MongoDB-Analyzer/

Build Log: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=61dc5ec941accb3633b447be

Notes:

I use the MongoDB Drivers Logo as the Logo for the MongoDB Analyzer documentation. I do not think the MongoDB Analyzer has a logo at this time, so this seemed like the best option as a logo for the docs. If Drivers is alright with this, I will propose this logo for the docs to the MongoDB Analyzer team.

### CODE REVIEW

**Review by terakilobyte — CHANGES_REQUESTED** [review]
I'd like to see the links reworked. I don't think it's ever a good idea to point to a URL we know will be redirected, such as any base URL for a versioned docset (docs.mongodb.com/x/ -> docs.mongodb.com/x/current)

File: source/develop-applications.txt [hunk 1]
```diff
@@ -68,3 +73,12 @@ distributed database built for application developers.
 
       Discover the mobile database & tools to build serverless apps, sync
       across devices, and authenticate users.
+
+   .. card::
+      :headline: MongoDB Analyzer
+      :url: https://docs.mongodb.com/mongodb-analyzer/
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/develop-applications.txt:79]: I'll make the edit!

The `//current` redirect most likely has to do with a cached redirect [from the issue fixed in this PR](https://github.com/mongodb/docs-visual-studio-extension/commit/0631bb7b599db28f30c4a61b0e219c811e497795). As no customers saw the docs set while this faulty redirect was present, we opted to not include a second redirect to map `//current` to `/current`

File: source/images/icons/analyzer.svg [hunk 2]
```diff
@@ -0,0 +1 @@
..."/><stop offset="1" stop-color="#80d8a4"/></linearGradient><linearGradient id="linear-gradient-2" x1="200.69" y1="413.64" x2="217.96" y2="413.64" gradientTransform="translate(-431.4 -138.28) rotate(-45)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#007daf"/><stop offset="0.11" stop-color="#0982b0"/><stop offset="0.29" stop-color="#2291b5"/><stop offset="0.51" stop-color="#4ba9bb"/><stop offset="0.77" stop-color="#83cac4"/><stop offset="1" stop-color="#b9eacd"/></linearGradient><linearGradient id="linear-gradient-3" x1="201.95" y1="430.91" x2="216.25" y2="419.53" xlink:href="#linear-gradient-2"/></defs><g id="Layer_1" data-name="Layer 1"><path class="cls-1" d="M9.34,14.66a10.65,10.65,0,0,1,3.05,8.83A11.49,11.49,0,0,1,.51,11.61,10.65,10.65,0,0,1,9.34,14.66Z"/><path class="cls-2" d="M16.57,1.44l-4.2,4.21a.79.79,0,0,0,0,1.13l1.85,1.86L9.89,13A12.26,12.26,0,0,0,.68,10a11.39,11.39,0,0,1,3.19-6.1A11.51,11.51,0,0,1,16.57,1.44Z"/><path class="cls-3" d="M22.44,7.18A11.5,11.5,0,0,1,14,23.32a12.26,12.26,0,0,0-3-9.21l4.33-4.33L17,11.45a.8.8,0,0,0,1.14,0Z"/><path class="cls-4" d="M20.13,3.87a10.79,10.79,0,0,1,1.5,1.84l-4,4L14.07,6.22l4-4A11.06,11.06,0,0,1,20.13,3.87Z"/></g></svg>
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/images/icons/analyzer.svg:1]: Will run this by Max

File: source/develop-applications.txt [hunk 3]
```diff
@@ -26,6 +26,11 @@ Develop Applications
       :project-name: realm
       :primary:
 
+   .. entry:: MongoDB Analyzer
+      :url: https://docs.mongodb.com/mongodb-analyzer/
```
[the remark this answers is not in the document]
Comment (biniona-mongodb) [reply source/develop-applications.txt]: https://github.com/mongodb/docs-landing/pull/82#discussion_r781346243

**Review by terakilobyte — APPROVED** [review]
lgtm

**Comment by biniona-mongodb** [comment]
Max Marcon and Shelby Carpenter approved on Slack with edits (that were taken [here](30f3e3c)).

**Pull request merged**